build.gradle

```diff
@@ -31,1271 +9,45 @@ plugins {
     id 'de.undercouch.download' version '5.4.0'
     id 'net.minecraftforge.gradle.patcher' version '[6.0.16,6.2)' apply false
     id 'net.minecraftforge.gradle.mcp' version '[6.0.16,6.2)' apply false
+    id 'net.minecraftforge.gradlejarsigner' version '1.0.4'
+    id 'org.barfuin.gradle.taskinfo' version '2.1.0'
 }
 
 Util.init() //Init all our extension methods!
 
 ext {
-    JAR_SIGNER = null
-    if (project.hasProperty('keystore')) {
-        JAR_SIGNER = [
-            storepass: project.properties.keystoreStorePass,
-            keypass: project.properties.keystoreKeyPass,
-            keystore: project.properties.keystore
-        ]
-    }
-    MAPPING_CHANNEL = 'official'
-    MC_VERSION = '1.20.2'
-    MAPPING_VERSION = MC_VERSION
-    MC_NEXT_VERSION = '1.21'
-    MCP_VERSION = '20230921.100330'
-    SNAPSHOT = false
-
-    INSTALLER_VERSION = '2.1.+'
-
     GIT_INFO = gradleutils.gitInfo
     VERSION = gradleutils.getFilteredMCTagOffsetBranchVersion(true, '[0-9]', MC_VERSION)
-    // FML_VERSION = gradleutils.getFilteredMCTagOffsetBranchVersion(true, 'FML', MC_VERSION)
     FORGE_VERSION = VERSION.substring(MC_VERSION.length() + 1)
 
-    BINPATCH_TOOL = 'net.minecraftforge:binarypatcher:1.1.1:fatjar'
-    INSTALLER_TOOLS = 'net.minecraftforge:installertools:1.3.0'
-    JAR_SPLITTER = 'net.minecraftforge:jarsplitter:1.1.4'
-    FART = 'net.minecraftforge:ForgeAutoRenamingTool:1.0.5:all'
-    MIN_TAG_FOR_CHANGELOG = '47.999'
+    BINPATCH_TOOL = 'net.minecraftforge:binarypatcher:1.2.0:fatjar'
+    INSTALLER_TOOLS = 'net.minecraftforge:installertools:1.4.0'
+    FART = 'net.minecraftforge:ForgeAutoRenamingTool:1.0.6'
 }
 
-println('Version: ' + VERSION +
-        ' Java: ' + System.getProperty('java.version') +
-        ' JVM: ' + System.getProperty('java.vm.version') + '(' + System.getProperty('java.vendor') + ')' +
-        ' Arch: ' + System.getProperty('os.arch'))
-
 changelog {
-    fromTag MIN_TAG_FOR_CHANGELOG
+    fromTag '47.999'
```

Comment:
Possible test change that you forgot to change back?

Reply:
Naw, it was a cleanup for the changelog intermingling. I can remember exactly what it was for but the changelog tasks was being stupid.